src/CozyUtils.js

```diff
@@ -146,6 +146,33 @@ class CozyUtils {
     }
   }
 
+  async createShortcuts(files) {
```

Comment:
This has nothing to do with creation here. You only manipulate data. 

Something like `getSplitedSchoolAndFav` or `computeShortcuts` or something, but not "create"

Reply:
Ok i get it, no problems !